examples/with-imagekit/pages/index.tsx

```diff
@@ -0,0 +1,115 @@
+import Head from "next/head";
+import { useEffect, useState } from "react";
+import { listFiles } from "@/utils/imagekit";
+import Masonry from "react-responsive-masonry";
+import styles from "@/styles/Home.module.css";
+import { FilterType } from "@/utils/types";
+import { FilterEnum } from "@/utils/enum";
+import { FileObject } from "imagekit/dist/libs/interfaces";
+import { IKImage, IKVideo } from "imagekit-next";
+import Link from "next/link";
```

Comment:
There's a ESLint plugin to organize imports. Please use it.

https://eslint.org/docs/latest/rules/sort-imports

Reply:
In the [guidelines](https://github.com/vercel/next.js/blob/canary/contributing/examples/adding-examples.md), next.js have mentioned that "Examples should not add custom ESLint configuration"
![Uploading Screenshot 2024-08-09 at 2.50.00 PM.png…]()